packages/elements/list/src/normalizers/normalizeListItem.ts

```diff
@@ -90,6 +94,26 @@ export const normalizeListItem = (
       type: getPlatePluginType(editor, ELEMENT_LIC),
     })
   ) {
+    if (
+      match(firstLiChildNode as any, {
+        type: getListTypes(editor),
+      })
+    ) {
+      // the listItem has no lic so we move the children up a level
+      const parent = getParent(editor, listItem[1]);
+      const sublist = firstLiChild;
+      const children = getChildren(firstLiChild).reverse();
+      Editor.withoutNormalizing(editor, () => {
```

Comment:
withoutNormalizing, on the other hand, is not needed in normalizations (the other call should go too).

Reply:
you are right `normalizeNode` is always called from a `withoutNormalizing`, I'll remove them